js/src/logger.ts

```diff
@@ -979,6 +979,7 @@ type InitLoggerOptions<IsAsyncFlush> = {
   apiUrl?: string;
   apiKey?: string;
   orgName?: string;
+  forceLogin?: boolean;
```

Comment:
Shoved this in too, so that our unit tests which exercise the lazy-login functionality could force a login.